tests/container_test.py

```diff
@@ -177,5 +177,9 @@ def test_container_version_label_matches(main_container):
         exec(f.read(), pkg_vars)  # nosec
     project_version = pkg_vars["__version__"]
     assert (
-        main_container.labels["version"] == project_version
+<<<<<<< HEAD
+        main_container.labels["org.opencontainers.image.version"] == project_version
+=======
+        version_container.labels["org.opencontainers.image.version"] == project_version
+>>>>>>> 9e993fbb4bbad507742670c91d98806190f37f0f
```

Comment:
And here too.

Reply:
Good catch.   Not sure how I did this, but everything on my local repo is good.  Re-force pushing...